pinot-query-runtime/src/test/resources/queries/TimeTypes.json

```diff
@@ -0,0 +1,72 @@
+{
+  "ts_without_tz": {
+    "tables": {
+      "ts": {
+        "schema": [
+          {"name": "data", "type": "TIMESTAMP"}
+        ],
+        "inputs": [
+          ["1999-01-08 04:05:06"],
+          ["1999-01-08 22:05:46"],
+          ["1999-01-08 04:05:06.001"],
+          ["5760-01-01 04:05:06"]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "psql": "8.5.1.3",
+        "description": "should support timestamps with various values",
```

Comment:
I thought we don't support time and byte types yet since there are a lot of functions we don't support yet.

@walterddr We were saying earlier we only support primitive types + string for v2 engine now. Does this narrative change?